gpii/node_modules/flowManager/test/configs/gpii.flowManager.tests.capture.config.json5

```diff
@@ -0,0 +1,4 @@
+{
+    "type": "gpii.flowManager.tests.capture.config",
```

Comment:
This config file doesn't seem to be used. It would be nice to have integration tests testing the capture tool with trusted and untrusted configs as what this config is intended for.

Reply:
I removed this unused config file, and added a comment on the other config that there are only untrusted configs for this functionality. (ie. it doesn't make sense to run a system capture for settings on a cloud server. This is just for users who are getting captures off of their desktop/laptops with an untrusted flowmanager running locally.)